TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -423,9 +423,6 @@ TYPO3:
       'TYPO3.Neos:Preview':
         maximumWidth: 1000
         maximumHeight: 1000
-      'TYPO3.Neos:IconPreview':
-        maximumWidth: 16
-        maximumHeight: 16
```

Comment:
Ok, you removed the usage above. Hm, then I don't get the usecase for the presets. I thought they would pregenerate thumbnails, and 16x16 is needed for the AssetList in the inspector…